README.md

```diff
@@ -143,6 +146,10 @@ with the KVM hypervisor driver.
   to use a different command to add `oc` to the _PATH_ environment variable.
   To verify the `oc` version, check the contents of the `~/.minishift/cache/oc` directory.
 
+For general information about using the `oc` command and the command-line console,
+see [Get Started with CLI](https://docs.openshift.org/latest/cli_reference/get_started_cli.html) in
```

Comment:
For general information about using the oc command and the command-line console...> This should probably be - ... about using the oc command-line and the web console. 

...see Get Started with CLI in the OpenShift documentation > The link provided here mostly talks about downloading OC CLI in various Operating systems, which afaiu is exactly what Minishift does for you in the 2 steps of the Quickstart. Also, the link solely talks about OC CLI, there is no information about web console here, so we could either drop reference to the console totally or add another link for information about web console. 

My 2 cents: A more relevant link in this section would probably be 'https://docs.openshift.org/latest/cli_reference/basic_cli_operations.html#cli-reference-basic-cli-operations', which would help Developers explore OC further once Minishift launches an OC instance for them. I think that would be the logical next-step for the Dev-user.